receiver/jmxreceiver/integration_test.go

```diff
@@ -43,6 +43,7 @@ import (
 var jmxJarReleases = map[string]string{
 	"1.0.0-alpha": "https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-java-contrib-jmx-metrics/1.0.0-alpha/opentelemetry-java-contrib-jmx-metrics-1.0.0-alpha.jar",
 	"1.4.0-alpha": "https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-jmx-metrics/1.4.0-alpha/opentelemetry-jmx-metrics-1.4.0-alpha.jar",
+	"1.6.0-alpha": "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v1.6.0/opentelemetry-jmx-metrics.jar",
```

Comment:
The tested script isn't compatible w/ this release since it still uses labels: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.36.0/receiver/jmxreceiver/testdata/script.groovy#L17

Reply:
@anuraaga I took a crack at updating this: https://gist.github.com/rmfitzpatrick/696bb9a8b236db0165b2359c898a70bb

Reply:
Thanks a lot! Do you mind if I close this PR and let you make it given you're the expert? :-)

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5429